db/schema.rb

```diff
@@ -48,6 +48,10 @@
     t.integer "solved_decrement_period", default: 1
     t.boolean "design_phase", default: false
     t.bigint "game_id"
+    t.boolean "sponsored", default: false, null: false
```

Comment:
This change is already merged into master, might need to rebase your branch and re-run your migrations to fix the schema.rb